src/components/mount/Mount.js

```diff
@@ -0,0 +1,50 @@
+import './mount.scss';
+
+export class Mount extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      status: 'init',
+      color: 'grey'
+    };
+
+    this.updateStatus = this.updateStatus.bind(this);
+  }
+
+  static getDerivedStateFromProps(nextProps) {
+    if (nextProps.code !== 0 && nextProps.code % 3 === 0 && nextProps.code % 5 === 0) {
+      return { color: '#0ff' };
+    }
+    if (nextProps.code !== 0 && nextProps.code % 3 === 0) {
+      return { color: 'green' };
+    }
+    if (nextProps.code !== 0 && nextProps.code % 5 === 0) {
+      return { color: 'blue' };
+    }
```

Comment:
I see 7 times repeated `nextProps.code`. Move it to separate variable or use destructuring.